src/fe-gtk/setup.c

```diff
@@ -2132,6 +2132,7 @@ setup_apply_real (int new_pix, int do_ulist, int do_layout, int do_identd)
 
 	mg_apply_setup ();
 	tray_apply_setup ();
+	hexchat_reinit_timers ();
```

Comment:
The preference variable used in `hexchat_check_lag()`, it is only exposed via `/set` which means it now cannot be changed without restarting. It really isn't your fault but I think ideally this would just be exposed in the preferences window.